go.rst

```diff
@@ -321,7 +299,32 @@ Wahl stehen.
      die restlichen Plätze nach Anzahl der Stimmen in der ersten Runde
      besetzt. Bei Gleichstand entscheidet das Los.
 
-9. Abwahlen sind auch bei Abwesenheit der betroffenen Person möglich und
+7.2  Der so bestimmten Gruppe muss anschließend das Vertrauen ausgesprochen werden. Dies geschieht 	falls nicht anders gewünscht per Akklamation.
+     Sind die ersten sechs Personen genannter Gruppe vom gleichen Geschlecht,
+     ersetzt die Person eines anderen Geschlechts mit den meisten Stimmen die
+     sechste Person in der Rangfolge.
+     Sollten sich nur Personen eines Geschlechts beworben haben, ist diese
+     Regelung irrelevant.
+
+     Bei weniger als sieben sich bewerbenden Speziemen der Gattung homo sapiens sapiens muss der
```

Comment:
```suggestion
     Bei weniger als sieben sich bewerbenden Personen muss der
```
Wir sprechen an allen Stellen von Personen, dann sollten wir das auch hier tun. Außerdem ist die Spezifizierung auf Homo Sapiens Sapiens speziesistisch, schließt unter anderen alle vergangenen und zukünftigen Evolutionsstufen des Menschen von der Gruppe der Vertrauenspersonen aus und widerspricht damit unserer eigenen Antidiskriminierungsklausel.

Reply:
ich hab das jetzt auch nochmal geändert, kommen wir uns dadurch in die Quere?